code-challenges.test.js

```diff
@@ -1,5 +1,14 @@
 // ASSESSMENT 2: Coding Practical Questions with Jest
 
+const { getConsoleOutput } = require("@jest/console")
+const { expect } = require("@jest/globals")
+const { conditionalExpression } = require("babel-types")
+const { create } = require("combined-stream")
+const { array } = require("yargs")
+
+// const { jest } = require("@jest/globals")
+// const { object } = require("webidl-conversions")
+
```

Comment:
These imports get added in automatically in certain text editors. You can remove them, they aren't helping anything. :)